gc/base/standard/Scavenger.cpp

```diff
@@ -4311,6 +4537,9 @@ MM_Scavenger::mutatorFinalReleaseCopyCaches(MM_EnvironmentBase *envBase, MM_Envi
 {
 	MM_EnvironmentStandard *env = MM_EnvironmentStandard::getEnvironment(envBase);
 	MM_EnvironmentStandard *threadEnvironment = MM_EnvironmentStandard::getEnvironment(threadEnvironmentBase);
+//	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
```

Comment:
Did you mean to leave this commented code in here?  If so could it be wrapped under a DEBUG if defined or something.